integration-tests/bulkExportTestHelper.ts

```diff
@@ -0,0 +1,116 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+/* eslint-disable class-methods-use-this */
+// eslint-disable-next-line import/no-extraneous-dependencies
+import axios, { AxiosInstance } from 'axios';
+import sampleBundle from './sampleBundle.json';
+
+export default class BulkExportTestHelper {
+    THREE_MINUTES_IN_MS = 3 * 60 * 1000;
+
+    fhirUserAxios: AxiosInstance;
+
+    constructor(fhirUserAxios: AxiosInstance) {
+        this.fhirUserAxios = fhirUserAxios;
+    }
+
+    async startExportJob(startExportJobParam: StartExportJobParam) {
+        try {
+            const params: any = {
+                _outputFormat: 'ndjson',
+            };
+            if (startExportJobParam.since) {
+                // eslint-disable-next-line no-underscore-dangle
+                params._since = startExportJobParam.since.toISOString();
+            }
+            if (startExportJobParam.type) {
+                // eslint-disable-next-line no-underscore-dangle
+                params._type = startExportJobParam.type;
+            }
+            const urlSearchParams = new URLSearchParams(params);
+            const response = await this.fhirUserAxios.get(`/$export?${urlSearchParams.toString()}`);
```

Comment:
axios can take `params`. There's no need to manually build the url params string
```js
axios.get('/user?ID=12345')
// same as
axios.get('/user', {
    params: {
      ID: 12345
    }
  })
```

Reply:
updated